azkaban-common/src/main/java/azkaban/metrics/ContainerMetrics.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.metrics;
+
+import azkaban.utils.Props;
+
+/**
+ * Defines all the metrics emitted by containerized executions
+ */
+public interface ContainerMetrics {
```

Comment:
Here we're defining metrics for Pod, Container and App Container. Are those terms interchangeable? If not, if they in fact different components should we generalize the name of this class to something like `ContainerizedExecutionMetrics` or `ContainerizationMetrics`

Reply:
They are not interchangeable. Rename the class to ContainerizationMetrics.